### PR TITLE
Update to freedesktop-sdk 19.08

### LINKED ISSUE #8: Update runtime to 19.08
I have already tried updating the runtime to 19.08, unfortunately, BlueJeans did not work after the update:
```
/app/extra:/app/extra/lib
/app/extra/bluejeans: error while loading shared libraries: libudev.so.0: cannot open shared object file: No such file or directory
```
Adding the _gudev_ shared-module did not help.

### PR DESCRIPTION
**Pull request by Erick555**
Fixes https://github.com/flathub/com.bluejeans.BlueJeans/issues/8

### CODE REVIEW

**Comment by Erick555** [comment]
bot, build

File: com.bluejeans.BlueJeans.yaml [hunk 1]
```diff
@@ -65,6 +65,7 @@ modules:
           - mv opt/bluejeans/* .
           - rm -r opt
           - chmod 755 bluejeans
+          - ARCH_TRIPLE=$(gcc --print-multiarch) && ln -s /usr/lib/${ARCH_TRIPLE}/libudev.so.1 /app/lib/libudev.so.0
```
Comment (AsciiWolf) [inline com.bluejeans.BlueJeans.yaml]: I may be wrong, but I don't think that using symlinks inside a Flatpak is a good idea.

Comment (Erick555) [reply com.bluejeans.BlueJeans.yaml]: Why? This is common solution.

Comment (Erick555) [reply com.bluejeans.BlueJeans.yaml]: Tons of examples: https://github.com/search?q=org%3Aflathub+ln+-s&type=Code

Comment (AsciiWolf) [reply com.bluejeans.BlueJeans.yaml]: Hmm, I thought that absolute paths in symlinks can be a problem for Flatpak. It seems that I was mistaken. @barthalion What is your opinion?

**Comment by Erick555** [comment]
@AsciiWolf this works now, ready for merge.

File: com.bluejeans.BlueJeans.yaml (hunk 1 above)
Comment (AsciiWolf) [reply com.bluejeans.BlueJeans.yaml]: I will merge the PR after this is approved by barthalion or other Flathub staff member.

**Comment by AsciiWolf** [comment]
Thanks!

**Pull request merged**